### ISSUE TITLE
Alt text missing

### ISSUE THREAD

**Issue by roelfrenkema** [ihabunek/toot#283]
Alt text is missing. This is an important feature that needs to be implemented.

**Comment by ihabunek** [comment]
Missing where?

**Comment by roelfrenkema** [comment]
Sorry my bad
